### DISCUSSION TITLE
Shadowsocks not working on IPv6-only environment

### DISCUSSION THREAD

**Discussion by okudayukiko** [shadowsocks/shadowsocks-android#2789] [General]
Server: IPv6-only VPS (no IPv4)
Shadowsocks-libev 3.2.5

PC (Windows 10 and Firefox):
Shadowsocks-windows 4.4.0
SSH with dynamic port forwarding: works (can open www.google.com)
Shadowsocks-windows: not works
Shadowsocks-windows (execute `pkill ss-server` and `ss-server -p port -k password -s ::0 -6` on Server): works

Shadowsocks-android 5.2.5:
Open www.google.com: not works
Open ipv6.google.com: works
Open www.debian.org: not works

**Comment by pexcn** [comment]
same issue

**Comment by profection** [comment]
shadowsocks是支持仅ipv6网络的
你需要在/etc/default/shadowsocks-libev中配置服务器端shadowsocks用ipv6优先解析dns
![image](https://github.com/shadowsocks/shadowsocks-android/assets/29281295/6bd1c023-4cde-43c8-b115-7a1fc08721fc)

**Reply by profection** [reply]
如果想在配置文件写，写这个：
"ipv6_first":true
